cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/MonitorConfig.java

```diff
@@ -278,6 +278,14 @@ public final class MonitorConfig {
   public static final String LINEAR_REGRESSION_MODEL_MIN_NUM_CPU_UTIL_BUCKETS_DOC = "The minimum number of full CPU"
       + " utilization buckets required to generate a linear regression model.";
 
+  /**
+   * <code>fetch.metric.samples.retry.count.max</code>
+   */
+  public static final String FETCH_METRIC_SAMPLES_RETRY_COUNT_MAX_CONFIG = "fetch.metric.samples.retry.count.max";
```

Comment:
Nit: from the other config naming convention, it seems "fetch.metric.samples.max.retry.count" is a better config name. Can we change that?